src/components/elements/carousel/carousel-style.js

```diff
@@ -135,12 +135,6 @@ export const NavigationContainer = styled(Flex)`
                 width: 100%;
                 height: ${height ?? '8px'};
                 bottom: ${bottom_offset ?? '40px'};
-                padding-left: 16px;
-                justify-content: left;
-
-                @media screen and (min-width: 785px) {
-                    padding-left: 102px;
-                }
             `
```

Comment:
Aren't this affecting other pages?

Reply:
no , it was added by @aswathy-deriv in this https://github.com/binary-com/deriv-com/pull/2998/files , she said she made the changes for academy, so I refactored it so that the styles are only specific to academy